STORY-7212 remove affiliate ringpool docs: removed all references to …

luanzeba opened the pull request:
…affiliate ringpools from network integration api

Commit:
STORY-7212 remove affiliate ringpool docs: removed all references to affiliate ringpools from network integration api

Commit:
STORY-7212 remove affiliate ringpool docs: removed all references to affiliate ringpools from network integration api

luanzeba commented:
Ready for code review. I only updated the latest version at first. Once the reviewer is happy with the changes, I will go back and apply to previous versions as well

codyjca commented:
@luan-vieira looks great!  A couple questions

Should we remove this section?
https://github.com/Invoca/developer-docs/blob/1904/story-7212_remove_affiliate_mentions_to_affiliate_ringpools/source/basics/error_handling.rst#affiliate-campaign-ringpools

What about the line `With the RingPool feature enabled, set up search campaigns to redirect to your landing page and search promo numbers are automatically displayed on the landing page` on this page? https://github.com/Invoca/developer-docs/blob/1904/story-7212_remove_affiliate_mentions_to_affiliate_ringpools/source/web_integration/publisher_web_integration.rst#publisher-web-integration

Lastly, can you publish a private version so we can view it in the portal? Just make sure formatting is correct?

When I have had to update all the versions I had to create PR's for each and tag an admin to get it approved, it was a complete pain. If you have to do that, I'd recommend:

- Squash all your changes into one commit
- Sit with someone with admin rights (or ask for admin rights) and pull each branch and cherry pick that single commit and push.  You are making changes to the branches directly but its a lot faster. Feel free to run that by someone else.

Commit:
STORY-7212 remove affiliate ringpool docs: removed publisher WIC page and other references to affiliate ringpools. Also replaced any mention of advertiser campaign ringpool to just ringpool

luanzeba commented:
thanks for catching those @codyjca , I removed the stuff you pointed out.
* I also changed any mention of "Advertiser Campaign RingPool" to say just "RingPool", since there's no distinction anymore and none of our brand customers think about "Advertiser Campaigns"
* Thank you for your suggestion to squash the commits and cherry pick them for previous versions by pairing with someone with admin rights, I will do that for sure!
* Lastly, the branch name in this PR was causing issues serving some content because of the `/` in the name, so I opened a new PR with a simpler branch name here. https://github.com/Invoca/developer-docs/pull/212
Can we move the conversation to there? I will close out this PR
